File: code-coaches/easy/balconies/Balconies.java
import java.util.Scanner;

public class Balconies {

    public static int calculateArea(String balcony) {
        String[] dimensions = balcony.split(",");
        int length = Integer.parseInt(dimensions[0]);
        int width = Integer.parseInt(dimensions[1]);
        return length * width;
    }

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String balconyA = input.nextLine();
        String balconyB = input.nextLine();

        int areaA = calculateArea(balconyA);
        int areaB = calculateArea(balconyB);

        if (areaA > areaB) {
            System.out.println("Apartment A");
        } else {
            System.out.println("Apartment B");
        }
    }

}

File: code-coaches/easy/balconies/balconies.rb
def calculate_area(balcony)
  dimensions = balcony.split(",")
  length = dimensions[0].to_i
  width = dimensions[1].to_i
  area = length * width
end

def main
  balcony_a = gets.chomp
  balcony_b = gets.chomp

  area_a = calculate_area(balcony_a)
  area_b = calculate_area(balcony_b)

  if area_a > area_b
    puts 'Apartment A'
  else
    puts 'Apartment B'
  end
end

if __FILE__ == $0
  main()¡¡
end
